cmd/kubeadm/app/apis/kubeadm/v1beta2/conversion.go

```diff
@@ -22,7 +22,14 @@ import (
 )
 
 func Convert_kubeadm_InitConfiguration_To_v1beta2_InitConfiguration(in *kubeadm.InitConfiguration, out *InitConfiguration, s conversion.Scope) error {
-	return autoConvert_kubeadm_InitConfiguration_To_v1beta2_InitConfiguration(in, out, s)
+	if err := autoConvert_kubeadm_InitConfiguration_To_v1beta2_InitConfiguration(in, out, s); err != nil {
+		return err
+	}
+
+	out.PublicKeyAlgorithm = &PublicKeyAlgorithm{
+		PublicKeyAlgorithm: in.PublicKeyAlgorithm,
```

Comment:
v1beta1 and v1beta2 are locked at this point, which means we can only add this as part of v1beta3.
i'm marking https://github.com/kubernetes/kubeadm/issues/1535 as `kind/api-change` so we need to batch include it in the next iteration of the API.

this is a nice change to push the kubeadm config forward.